Demo/Demo/FeatureViewControllers/PaymentButtonCustomizationViewController.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+import PayPalUI
+
+class PaymentButtonCustomizationViewController: UIViewController {
+
+
+    // MARK: - Views
+
+    /// Scroll view which will auto size
+    let scrollView = UIScrollView(frame: .zero)
+
+    lazy var paymentButton: PaymentButton = setupPaymentButton()
+
+    lazy var spbFundingPicker: UISegmentedControl = {
+        let segment = UISegmentedControl(
+            items: PaymentButtonFundingSource.allCases.map { $0.rawValue }
+        )
+        segment.selectedSegmentIndex = 0
+        segment.addTarget(self, action: #selector(reloadSmartPaymentButton), for: .valueChanged)
+        return segment
+    }()
+
+    lazy var spbColorPicker: UISegmentedControl = {
+        let segment = UISegmentedControl(items: [])
+        segment.addTarget(self, action: #selector(reloadSmartPaymentButton), for: .valueChanged)
+        return segment
+    }()
+
+    lazy var spbEdgesPicker: UISegmentedControl = {
+        let segment = UISegmentedControl(
+            items: PaymentButtonEdges.allCases.map { $0.description }
+        )
+        segment.selectedSegmentIndex = 1
+        segment.addTarget(self, action: #selector(reloadSmartPaymentButton), for: .valueChanged)
+        return segment
+    }()
+
+    lazy var spbSizePicker: UISegmentedControl = {
+        let segment = UISegmentedControl(
+            items: PaymentButtonSize.allCases.map { $0.description }
+        )
+        segment.selectedSegmentIndex = 1
+        segment.addTarget(self, action: #selector(reloadSmartPaymentButton), for: .valueChanged)
+        return segment
+    }()
+
+    lazy var stackView: UIStackView = {
+        let stackView = UIStackView(
+            arrangedSubviews: [
+                label("SmartPaymentButton Configuration"),
+                spbFundingPicker,
+                spbColorPicker,
+                spbEdgesPicker,
+                spbSizePicker
+            ]
+        )
+        stackView.axis = .vertical
+        stackView.alignment = .leading
+        stackView.spacing = 16.0
+        stackView.distribution = .fill
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(
+            top: 16.0,
+            left: 16.0,
+            bottom: 16.0,
+            right: 16.0
+        )
+        return stackView
+    }()
+
+    // MARK: - Init
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        configure()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
If we override `viewDidLoad` these shouldn't be needed.

Reply:
i tried it but it crashes :(

Reply:
i think there is a conflict in rendering, its like its trying to do some stuff when the views havent been stacked or anything yet.